Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -235,9 +235,8 @@ class TabTrayController: UIViewController {
     private var privateMode: Bool = false {
         didSet {
             if #available(iOS 9, *) {
-                togglePrivateMode.selected = privateMode
+                togglePrivateMode.selected = !togglePrivateMode.selected
```

Comment:
This change seems weird...why wouldn't we want `togglePrivateMode.selected` to always match `privateMode`?